src/routers/controllers/removeCompanyController.ts

```diff
@@ -0,0 +1,32 @@
+import { Request, Response } from "express";
```

Comment:
This controller has low test coverage. The red numbers indicate lines of code not covered by tests at all.
![Screenshot 2024-10-03 at 11 24 54](https://github.com/user-attachments/assets/994a1da2-97c7-4766-8fb2-90abc591740c)

Reply:
Similar to above, we refactored to get rid unnecessary lines, could you re-review please?